BraveShared/BraveStrings.swift

```diff
@@ -117,6 +117,32 @@ extension Strings {
     public static let suppressAlertsActionMessage = NSLocalizedString("SuppressAlertsActionMessage", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Prevent this page from creating additional alerts", comment: "Message body of alert that seeks permission from user to suppress future JS alerts")
 }
 
+// MARK:-  DefaultBrowserIntroCalloutViewController.swift
+extension Strings {
+    public static let defaultBrowserIntroPrimaryText =
```

Comment:
I might better idea to use them under DefaultBrowser struct and also use it in key for localized string.